src/pyfmi/fmi3.pyx

```diff
@@ -1053,6 +1104,76 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
         cdef FMIL3.fmi3_string_t gen = <FMIL3.fmi3_string_t>FMIL3.fmi3_import_get_generation_tool(self._fmu)
         return pyfmi_util.decode(gen) if gen != NULL else ""
 
+    def get_variable_alias_base(self, variable_name):
+        """
+        Returns the base variable for the provided variable name.
+
+        Parameters::
+
+            variable_name--
+                Name of the variable.
+
+        Returns:
+
+           The base variable.
+        """
+        cdef FMIL3.fmi3_import_variable_t* variable
+        variable_name = pyfmi_util.encode(variable_name)
+        cdef char* variablename = variable_name
+
+        variable = FMIL3.fmi3_import_get_variable_by_name(self._fmu, variablename)
+        if variable == NULL:
+            raise FMUException("The variable %s could not be found."%variablename)
+
+        name = pyfmi_util.decode(FMIL3.fmi3_import_get_variable_name(variable))
+
+        return name
+
+    def get_variable_alias(self, variable_name):
+        """
+        Return a dict of all alias variables belonging to the provided variable
+        where the key are the names and the value indicati whether the variable
```

Comment:
```suggestion
        where the key are the names and the value indicates if the variable
```
Also, if the value indicates if it is an alias or not, doesn't that conflict with the intention of the function since it is supposed to return all alias variables belonging to the provided variable? I.e. all `values` would be `True`?
Maybe I just misunderstood this function.

Reply:
I suppose from a pure FMI3 perspective, this seems a little odd; but I think the final goal really is that an end-user can conveniently work with whatever alias of the eventual variable that they want and doesn't need to pay particular attention if it is the base or not. 

This particularly includes calling `fmu_get_variable_alias(<alias_name>)`